java/src/jmri/jmrit/automat/AutomatSummary.java

```diff
@@ -27,7 +27,7 @@ static public AutomatSummary instance() {
         return self;
     }
 
-    private final ArrayList<AbstractAutomaton> automats = new ArrayList<>();
+    public final ArrayList<AbstractAutomaton> automats = new ArrayList<>();
```

Comment:
Is there a reason this needs to be public?  The new `getAutomats()` method gives access to a copy.

Reply:
None. I was playing with synchronization outside of AutomatSummary